src/libs/TransactionUtils.js

```diff
@@ -253,6 +257,16 @@ function getCategory(transaction) {
     return lodashGet(transaction, 'category', '');
 }
 
+/**
+ * Return the billable from the transaction. This "billable" field has no "modified" complement.
+ *
+ * @param {Object} transaction
+ * @return {String}
+ */
+function getBillable(transaction) {
+    return lodashGet(transaction, 'billable', '');
```

Comment:
@waterim Shouldn't default value of billable be Boolean value, as we are using it for the toggle value? empty string falls back to the `false`

Reply:
Oups, yes, changing

Reply:
great, after your change I'll approve the PR 🚀

Reply:
changed 🙂